profiles/Jonas-setup.yaml

```diff
@@ -0,0 +1,218 @@
+---
+# Hunting Settings
+
+training_manager_hunting_priority: true
+training_manager_priority_skills:
+- Parry Ability
+hunting_info:
+- :zone: young_ogres
+  stop_on:
+  - Parry Ability
+  args:
+  - d0
+  - thanatology
+  :duration: 60
+
+- :zone: blue_crocs
+  stop_on:
+  - Skinning
+  args:
+  - d0
+  - skinning
+  :duration: 60
+
+weapon_training:
+  Crossbow: forester's crossbow
+  Brawling: ''
+  Small Edged: throwing axe
+  Offhand Weapon: throwing axe
+  Light Thrown: throwing axe
+
+offensive_spells:
+- skill: Targeted Magic
+  name: Acid Splash
+  abbrev: acs
+  mana: 1
+- skill: Debilitation
+  name: Petrifying Visions
+  abbrev: pv
+  mana: 1
+  harmless: true
+  expire: no longer seems petrified
+
+fatigue_regen_threshold: 85
+fatigue_regen_action: analyze
+use_stealth_attacks: true
+
+# Gear & Equipment
+
+gear:
+- :name: spike
+  :adjective: throwing
+  :is_leather: false
+- :name: axe
+  :adjective: throwing
+  :is_leather: false
+- :adjective: small
+  :name: shield
+  :is_leather: true
+  :hinders_lockpicking: true
+  :is_worn: true
+- :adjective: forester's
+  :name: crossbow
+  :is_worn: true
+  :is_leather: true
+- :adjective: quilted
+  :name: gloves
+  :is_leather: true
+  :hinders_lockpicking: true
+  :is_worn: true
+- :adjective: quilted
+  :name: hauberk
+  :is_leather: true
+  :hinders_lockpicking: true
+  :is_worn: true
+- :adjective: steel
+  :name: knuckles
+  :is_leather: false
+  :hinders_lockpicking: true
+  :is_worn: true
+- :adjective: elbow
+  :name: spikes
+  :is_leather: false
+  :is_worn: true
+- :adjective: knee
+  :name: spikes
+  :is_leather: false
+  :is_worn: true
+- :adjective: quilted
+  :name: mask
+  :is_leather: true
+  :hinders_lockpicking: true
+  :is_worn: true
+- :adjective: quilted
+  :name: hood
+  :is_leather: true
+  :hinders_lockpicking: true
+  :is_worn: true
+
+hand_armor: gloves
+
+gear_sets:
+  standard:
+  - small shield
+  - steel knuckles
+  - quilted gloves
+  - quilted hauberk
+  - quilted hood
+  - quilted mask
+  - elbow spikes
+  - knee spikes
+  - forester's crossbow
+  naked: []
+
+prep_scaling_factor: 0.95
+cambrinth: cambrinth anklet
+cambrinth_cap: 12
+held_cambrinth: false
+buff_spells:
+  Obfuscation:
+    abbrev: obf
+    recast: 1
+    mana: 19
+    cambrinth:
+      - 12
+  Manifest Force:
+    abbrev: maf
+    recast: 1
+    mana: 5
+    cambrinth:
+      - 12
+training_abilities:
+  Hunt: 80
+  App: 60
+  Stealth: 30
+
+skinning:
+  skin: true
+  arrange_count: 5
+  tie_bundle: true
+
+# Necromancer Related
+necromancer_healing:
+  CF:
+  - 15
```

Comment:
It's how empath healing is setup, which means we should update empath healing as well. And maybe unify training spells and combat spells.